### DISCUSSION TITLE
Deployment on AWS China Region

### DISCUSSION THREAD

**Discussion by ryanyangd** [SUSE/ha-sap-terraform-deployments#880] [Q&A] [answered]
Hi Team,

Could we use the code to deploy S4HANA in AWS China Regions(cn-north-1 or cn-northwest-1)?

Kind regards!
Ryan

**Comment by yeoldegrove** [comment, marked as answer]
@ryanyangd To be honest, I never tried to deploy it in the Chinese AWS regions.
It "should" work as far as I can judge.
Did you already try to do so and are facing any issues?
